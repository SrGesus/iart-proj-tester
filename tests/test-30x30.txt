VB	LV	LV	BB	LV	LH	VB	VC	FB	VE	BB	LH	VE	VB	LV	LV	LV	VD	FD	VB	BB	VD	FD	LH	VD	VD	LH	LH	BE	VD
LH	VD	FB	LH	VC	LH	BB	BE	VB	VC	VD	VD	BB	VB	FC	LH	LV	BE	VB	LH	LH	VE	LH	LV	BD	LV	FE	VC	VC	LV
VE	VE	VD	VB	LV	VE	VB	FC	LV	VE	LH	VB	FB	VB	LH	LV	LV	LV	LV	VC	VC	VB	FD	LH	BE	LH	LH	VC	VB	LH
VD	LH	VE	FC	VB	LV	VC	LH	VC	VC	VB	VC	VD	VE	LH	LV	LH	LH	LH	VB	FB	VB	LV	VD	FE	VB	BD	VE	LH	LV
BD	LH	LV	VD	VD	VD	VE	VD	VD	VE	VB	VC	VD	LV	LH	LH	VD	VC	VD	VC	BC	LV	VE	VC	LH	LV	VC	LH	LV	LV
VC	VD	FE	LH	VB	LV	VB	LH	FB	BD	VE	VD	LH	LV	LH	VD	VC	VD	VD	BC	VC	FD	LV	VC	LH	VC	VD	VE	LV	LV
FB	VD	BC	VD	LH	LH	LV	VE	VB	BB	VD	VD	FD	LV	LH	BB	VC	FE	VE	LV	VC	VE	LH	VE	VE	VE	VD	VB	VB	LH
BC	LH	VC	FC	LV	LH	LV	BB	VC	FB	VD	LV	LH	LH	LH	VC	VD	BB	VE	VE	BE	LH	VB	VD	VC	VD	VC	VB	VE	LV
VE	VB	FD	BB	LV	LH	VE	LH	VD	LV	BE	LH	LV	LV	VE	LH	VD	VD	VE	VB	VE	LH	FC	LV	VC	BD	LV	LH	VB	LH
VD	VB	VC	VE	FE	VD	VB	VE	BB	FC	VC	VE	FC	LH	BE	LH	VC	VD	LH	LH	VC	LV	LH	VE	LH	VB	LH	VC	FD	VD
VB	LV	VD	VE	VC	VE	LV	LH	LH	LV	VD	LH	VD	VE	FE	BE	VB	FB	LH	VC	VB	VE	VE	VC	VE	VD	VC	VE	LV	VC
FE	VB	VE	BD	BC	VD	FE	VB	LV	LV	VB	VD	VD	VC	VB	LV	VD	VE	VD	LV	VD	FE	LV	LV	VB	FC	VE	VD	FB	LH
VC	VD	BD	FC	LH	LH	LV	VC	LV	VD	VB	BB	VD	FB	LH	VE	VC	VB	BC	FC	VB	VD	LV	LV	VC	LH	VD	LV	LV	LV
BC	LV	VC	VB	VD	VE	BC	VD	FC	LV	FB	LH	VE	BD	VD	VC	VC	FC	VB	BE	VB	LV	VC	BB	LH	FC	VE	VD	LH	LH
LH	VB	LV	VB	VC	LV	VD	LV	LH	LV	LH	VC	VC	VE	FE	VB	LH	LH	VB	LV	LH	VB	VE	VE	LH	LH	LH	VD	BD	VB
FC	LV	VE	VE	VB	VD	LV	VD	BC	VC	BE	VD	LH	VB	VD	VB	LH	LH	BE	VE	VD	VE	LV	LH	FD	VB	VD	BD	LH	FC
VB	VD	LV	VD	VD	LH	VB	FD	VC	BD	FD	LV	VB	VE	VE	VC	VE	VB	FD	VE	VB	VB	VE	BE	LH	VC	LV	FE	LH	LV
VC	VE	VB	VC	LH	VB	LV	LH	LH	BD	VB	VC	FD	VC	LV	LH	VC	VB	VC	LH	VE	LV	LV	VB	VD	LV	VC	VE	VE	LH
VC	VB	VB	VE	VC	VD	VC	LV	VE	FE	VB	VE	LH	VD	VD	VB	LH	LV	VE	BE	LV	LH	LH	VC	LV	VC	VB	LH	VC	BE
BC	VB	VE	VE	VE	VB	LH	FB	VE	LH	LV	BE	BC	FC	LH	VE	VB	FC	LH	BD	BD	FD	VB	VE	VC	VB	LV	VB	VE	LH
LV	VB	LH	VD	VE	VD	VD	BE	VE	LH	VE	FB	VD	LV	VB	FB	BC	LH	LV	VC	FE	VB	VE	LV	VD	LH	BC	VC	LH	VB
VD	VD	VE	VC	VB	VC	VB	VC	BC	VD	BD	VE	VD	LH	LH	LH	LV	VC	VC	VC	VC	VC	FD	VB	VC	VB	VD	LV	VD	FE
VB	VC	LH	LH	VC	LV	VC	VB	VD	LV	FC	VB	BC	FE	VE	BE	FE	LH	VD	LH	VB	FE	BB	BE	LV	VD	VC	VE	BD	VD
LV	FB	BB	LV	FC	LV	VD	VC	VB	VB	LH	VD	VB	LH	BE	VE	VE	VD	LH	LH	VD	VD	VD	VB	LV	FE	LH	FD	LH	LV
VD	LH	VC	VE	LH	VD	LV	VC	VE	VB	VE	VE	LV	VD	BC	FE	VD	BB	LV	FB	LH	LV	VB	LV	LH	LV	VB	VD	VC	LH
FC	VE	VD	VC	VC	LH	BB	VB	FC	BC	FC	LH	FD	LV	LV	VE	VC	VE	LH	VC	VE	VE	VE	LH	FC	VB	LV	BD	LV	BE
LH	LH	VE	VE	VB	VE	LH	VD	LH	BD	VC	VC	BB	VD	FE	LV	VC	LH	VD	BE	LV	LV	BD	VC	LH	BB	VC	LH	FD	BB
LH	LH	VC	LV	LH	FE	BC	VC	FC	VE	LH	FB	LH	VE	LH	VB	VB	LV	VE	VC	LH	FB	BC	VC	VB	VE	FB	VE	VC	LH
LV	LV	VB	LH	LH	LH	VC	VB	LH	VB	LV	BE	VD	LV	VB	VB	VC	VC	FD	BD	LV	VD	VB	FD	LV	VD	LH	LH	VB	LV
VE	BC	LV	LV	LH	LV	LH	VD	FD	BB	VB	VE	LV	VD	FE	VD	LV	BC	LH	VB	FB	BC	LV	LH	VD	FD	LV	LV	LH	VD
