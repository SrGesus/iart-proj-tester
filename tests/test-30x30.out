VB	LH	LH	BB	LH	LH	VE	VB	FE	VB	BB	LH	VE	VB	LH	LH	LH	VE	FB	VB	BB	VE	FD	LH	VE	VB	LH	LH	BB	VE
LV	VB	FE	LV	VB	LH	BC	BE	VB	VC	VD	VE	BD	VC	FD	LH	LH	BC	VC	LV	LV	VD	LH	LH	BE	LV	FB	VB	VC	LV
VD	VC	VB	VC	LV	VB	VE	FC	LV	VB	LH	VC	FC	VB	LH	LH	LH	LH	LH	VC	VD	VE	FD	LH	BE	LV	LV	VD	VE	LV
VB	LH	VC	FD	VC	LV	VD	LH	VC	VD	VE	VB	VE	VD	LH	LH	LH	LH	LH	VE	FB	VD	LH	VE	FC	VD	BC	VE	LV	LV
BD	LH	LH	VE	VB	VC	VB	VE	VB	VE	VD	VC	VD	LH	LH	LH	VE	VB	VE	VD	BC	LH	VE	VD	LH	LH	VE	LV	LV	LV
VD	VE	FB	LV	VD	LH	VC	LV	FC	BD	VE	VB	LH	LH	LH	VE	VD	VC	VD	BB	VE	FB	LV	VB	LH	VE	VD	VC	LV	LV
FB	VD	BE	VD	LH	LH	LH	VC	VB	BE	VD	VC	FD	LH	LH	BC	VE	FD	VE	LV	VD	VC	LV	VD	VE	VD	VE	VB	VC	LV
BD	LH	VC	FD	LH	LH	LH	BB	VC	FC	VB	LH	LH	LH	LH	VE	VD	BB	VC	VD	BB	LH	VC	VB	VC	VB	VC	VD	VE	LV
VD	VE	FD	BB	LH	LH	VE	LV	VB	LH	BC	LH	LH	LH	VE	LV	VB	VC	VB	VE	VD	LH	FE	LV	VB	BC	LH	LH	VC	LV
VB	VC	VB	VC	FB	VB	VC	VD	BC	FE	VB	VE	FD	LH	BE	LV	VD	VE	LV	LV	VB	LH	LH	VC	LV	VB	LH	VE	FD	VC
VD	LH	VC	VB	VC	VD	LH	LH	LH	LH	VC	LV	VB	VE	FC	BD	VE	FC	LV	VD	VC	VB	VE	VB	VC	VD	VE	VD	LH	VE
FD	VE	VB	BC	BB	VE	FB	VB	LH	LH	VE	VD	VC	VD	VE	LV	VD	VE	VD	LH	VE	FC	LV	LV	VB	FE	VD	VE	FB	LV
VB	VC	BD	FE	LV	LV	LV	VD	LH	VE	VD	BB	VE	FB	LV	VD	VE	VD	BB	FE	VD	VE	LV	LV	VD	LH	VE	LV	LV	LV
BD	LH	VC	VB	VC	VD	BC	VE	FB	LV	FB	LV	VD	BE	VD	VE	VD	FE	VD	BB	VE	LV	VD	BC	LH	FE	VD	VC	LV	LV
LV	VB	LH	VC	VB	LH	VE	LV	LV	LV	LV	VD	VE	VD	FE	VD	LH	LH	VE	LV	LV	VD	VE	VB	LH	LH	LH	VE	BD	VC
FC	LV	VB	VE	VD	VE	LV	VD	BE	VD	BC	VE	LV	VB	VE	VB	LH	LH	BC	VC	VD	VE	LV	LV	FD	VE	VB	BE	LV	FB
VB	VC	LV	VD	VE	LV	VD	FE	VD	BB	FE	LV	VD	VC	VD	VC	VB	VE	FD	VE	VB	VC	VD	BC	LH	VC	LV	FC	LV	LV
VD	VE	VD	VE	LV	VD	LH	LH	LH	BE	VB	VC	FB	VB	LH	LH	VC	VD	VE	LV	VD	LH	LH	VE	VB	LH	VC	VB	VC	LV
VB	VC	VB	VC	VD	VE	VB	LH	VE	FC	VD	VE	LV	VD	VE	VB	LH	LH	VC	BD	LH	LH	LH	VC	LV	VB	VE	LV	VB	BE
BD	VE	VD	VE	VB	VC	LV	FB	VD	LH	LH	BC	BC	FE	LV	VD	VE	FD	LH	BC	BB	FE	VB	VE	VD	VC	LV	VD	VC	LV
LV	VD	LH	VC	VD	VE	VD	BE	VB	LH	VE	FB	VB	LH	VC	FD	BC	LH	LH	VE	FC	VB	VC	LV	VB	LH	BE	VB	LH	VC
VD	VE	VB	VE	VB	VC	VB	VC	BD	VE	BD	VC	VD	LH	LH	LH	LH	VE	VB	VC	VB	VC	FB	VD	VC	VB	VC	LV	VB	FE
VB	VC	LV	LV	VD	LH	VC	VB	VC	LV	FC	VB	BB	FE	VB	BB	FE	LV	VD	LH	VC	FD	BC	BB	LH	VC	VB	VC	BD	VE
LV	FD	BE	LV	FD	LH	VE	VD	VE	VD	LH	VC	VD	LH	BE	VD	VE	VD	LH	LH	VE	VB	VE	VD	LH	FE	LV	FB	LV	LV
VD	LH	VC	VD	LH	VE	LV	VB	VC	VB	VE	VB	LH	VE	BD	FE	VD	BB	LH	FE	LV	LV	VD	LH	LH	LH	VC	VD	VC	LV
FB	VB	VE	VB	VE	LV	BD	VC	FD	BE	FC	LV	FB	LV	LV	VB	VE	VD	LH	VE	VD	VC	VB	LH	FE	VB	LH	BB	LH	BE
LV	LV	VD	VC	VD	VC	LV	VB	LH	BE	VB	VC	BD	VC	FC	LV	VD	LH	VE	BD	LH	LH	BE	VB	LH	BC	VE	LV	FD	BE
LV	LV	VB	LH	LH	FE	BD	VC	FD	VC	LV	FB	LV	VB	LH	VC	VB	LH	VC	VD	LH	FE	BD	VC	VB	VE	FC	VD	VE	LV
LV	LV	VD	LH	LH	LH	VC	VB	LH	VE	LV	BD	VC	LV	VB	VE	VD	VE	FD	BB	LH	VE	VD	FE	LV	VD	LH	LH	VC	LV
VD	BC	LH	LH	LH	LH	LH	VC	FD	BC	VC	VD	LH	VC	FC	VD	LH	BC	LH	VC	FD	BC	LH	LH	VC	FD	LH	LH	LH	VC
